packages/cli/src/lib/implementation/config-middleware.spec.ts

```diff
@@ -6,38 +6,27 @@ import { configMiddleware } from './config-middleware';
 const __dirname = dirname(fileURLToPath(import.meta.url));
 const withDirName = (path: string) => join(__dirname, path);
 const config = (ext: string) =>
-  `${withDirName(
-    join(
-      '..',
-      '..',
-      '..',
-      '..',
-      'models',
-      'test',
-      'fixtures',
-      'code-pushup.config.mock.',
-    ),
-  )}${ext}`;
+  `${withDirName('../../../test/js-format.config.mock.')}${ext}`;
```

Comment:
Always use `path.join`, please :pray: 

```suggestion
  withDirName(
    join(
      '..',
      '..',
      '..',
      'test',
      `js-format.config.mock.${ext}`
    ),
  );
```

Reply:
Maybe create a simple `normalize` function
```ts
function normalize(path: string): string {
  return join(path.split('/'));
}
```

`join` looks bulky here.